### ISSUE TITLE
 Undefined variables in WordPress 5.4 beta

### ISSUE THREAD

**Issue by javiercasares** [audrasjb/wp-autoupdates#1]
I'm trying the plugin on the new version and found those errors

![Screenshot_57](https://user-images.githubusercontent.com/10290686/74809185-e5d62e80-52ec-11ea-819f-97d184c67773.png)

I found this when is not enabled on a plugin:

```
Notice: Undefined variable: plugin_name in /home/wpfuturo/public_html/wp-content/plugins/wp-autoupdates/wp-autoupdates.php on line 134
Notice: Undefined variable: page in /home/wpfuturo/public_html/wp-content/plugins/wp-autoupdates/wp-autoupdates.php on line 140
Enable automatic updates
```

And this when is enabled:

```
Notice: Undefined variable: plugin_name in /home/wpfuturo/public_html/wp-content/plugins/wp-autoupdates/wp-autoupdates.php on line 113
Automatic updates enabled
Notice: Undefined variable: page in /home/wpfuturo/public_html/wp-content/plugins/wp-autoupdates/wp-autoupdates.php on line 122
Disable
```

So, is not set the `plugin_name` and `page` variables.

Also, I found this message:

`Notice: Undefined index: action in /home/wpfuturo/public_html/wp-content/plugins/wp-autoupdates/wp-autoupdates.php on line 171`

**Comment by audrasjb** [comment]
Thanks for opening this issue, @javiercasares 👍
Partially fixed in #4

**Comment by pedro-mendonca** [comment]
Completely fixed in #5

**Comment by audrasjb** [comment]
Fixed in #5
